grpcbp/tracing.go

```diff
@@ -44,3 +44,13 @@ func methodSlug(method string) string {
 	split := strings.Split(method, "/")
 	return split[len(split)-1]
 }
```

Comment:
🔕 this function is likely no longer used?

Reply:
there is one other use of this method `methodSlug` in `MonitorInterceptorUnary`